source/guide_symfony.rst

```diff
@@ -0,0 +1,198 @@
+.. highlight:: console
+
+.. author:: Sascha Groetzner <https://groetzner.net>
+
+.. tag:: web
+.. tag:: lang-php
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/symfony.svg
+      :align: center
+
+##########
+Symfony
+##########
+
+.. tag_list::
+
+`Symfony <https://symfony.com/what-is-symfony>`_ is a set of PHP Components, a Web Application framework, a Philosophy, and a Community — all working together in harmony.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`domains <web-domains>`
+  * :manual:`PHP <lang-php>`
+
+License
+=======
+
+Symfony Source Code is published under the terms of the MIT License
+
+  * `<https://symfony.com/license>`_
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 8:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '8.0'
+ [isabell@stardust ~]$
+
+Your symfony URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Install symfony cli
+-------------------
+
+::
+
+ [isabell@stardust ~]$ wget https://get.symfony.com/cli/installer -O - | bash
+
+Modify ``~/.bash_profile`` to use symfony directly
+
+.. code-block:: ini
+
+ ...
+ # User specific environment and startup programs
+
+ PATH=$HOME/.local/bin:$HOME/bin:$PATH
+ PATH="$HOME/.symfony/bin:$PATH"
+ export PATH
+
+
+Create symfony project
+----------------------
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/isabell/
+ [isabell@stardust isabell]$ symfony new my_project
+ * Creating a new Symfony project with Composer
+   (running /usr/local/bin/composer create-project symfony/skeleton /var/www/virtual/isabell/my_project  --no-interaction)
+ * Setting up the project under Git version control
+   (running git init /var/www/virtual/isabell/my_project)
+ [OK] Your project is now ready in /var/www/virtual/isabell/my_project
+
+Adding Rewrite Rules
+
+::
+
+ [isabell@stardust isabell]$ composer require symfony/apache-pack
```

Comment:
Are you sure that this should be installed directly into `/var/www/virtual/isabell` instead of the `my_project` directory? This way we end up with two sets of composer files:
```
[oslides@pavo oslides]$ ll
total 8
-rw-rw-r--. 1 oslides oslides   65 Apr 19 09:58 composer.json
-rw-rw-r--. 1 oslides oslides 1652 Apr 19 09:58 composer.lock
drwxr-xr-x. 2 oslides oslides   28 Apr 19 09:40 html
drwxrwxr-x. 9 oslides oslides  185 Apr 19 09:58 my_project
drwxrwxr-x. 4 oslides oslides   57 Apr 19 09:58 vendor
[oslides@pavo my_project]$ ll
total 100
drwxrwxr-x. 2 oslides oslides    21 Apr 19 09:58 bin
-rw-rw-r--. 1 oslides oslides  1372 Apr 19 09:58 composer.json
-rw-rw-r--. 1 oslides oslides 91473 Apr 19 09:58 composer.lock
drwxrwxr-x. 4 oslides oslides   114 Apr 19 09:58 config
drwxrwxr-x. 2 oslides oslides    23 Apr 19 09:58 public
drwxrwxr-x. 3 oslides oslides    42 Apr 19 09:58 src
-rw-rw-r--. 1 oslides oslides  3540 Apr 19 09:58 symfony.lock
drwxrwxrwx. 4 oslides oslides    30 Apr 19 09:58 var
drwxrwxr-x. 6 oslides oslides    79 Apr 19 09:58 vendor
```